yt/config.py

```diff
@@ -1,20 +1,3 @@
-"""
-This module is very simple.  It imports the configuration
-we have written for yt.
-Everything will be returned in a global config dictionary ``ytcfg``
```

Comment:
Since you're removing these strings in addition to the copyright info, do we want to put anything in our developer guide that we don't recommend having header strings in new code to preserve style?

Reply:
Good point